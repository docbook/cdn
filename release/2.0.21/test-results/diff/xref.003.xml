<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: xref.003.xml</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="article" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: xref.003.xml</h2>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">Adam</span>
                     <span class="surname">Di Carlo</span>
                  </span>
               </h3>
            </div>
            <p class="CVS">$Id: xref.003.xml 4056 2004-11-23 13:28:50Z nwalsh $</p>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#xreftests">1. Xref test: titleabbrev</a>
                     </span>
                  </li>
                  <li>
                     <span>
                        <a href="#section">2. Cross-ref targets</a>
                     </span>
                  </li>
               </ul>
            </div>
         </div>
         <section deltaxml:deltaV2="A" id="xreftests" class="section">
            <div class="section-titlepage">
               <h2>1. Xref test: titleabbrev</h2>
            </div>
            <p>
      We're testing that xref uses the <code class="tag-element">titleabbrev</code>
      element where appropriate.
    </p>
            <p>
      Article: <a href="#article" title="Unit Test: xref.003.xml">Unit 03</a>.
    </p>
            <p>
      Section: <a href="#section" title="Cross-reference targets">Section 2, “Cross-ref targets”</a>.
    </p>
            <p>
      Figure: <a href="#figure" title="Long Title for Figure">Figure 1, “Short Fig Title”</a>.
    </p>
            <p>
      Example: <a href="#example" title="Long Title for Example">Example 1, “Short Examp Title”</a>.
    </p>
            <p>
      Table: <a href="#table" title="Long Title for Table">Table 1, “Short Table Title”</a>.
    </p>
         </section><div deltaxml:deltaV2="B" class="content">
            <section id="xreftests" class="section">
               <div class="section-titlepage">
                  <h2>1. Xref test: titleabbrev</h2>
               </div>
               <div class="content">
                  <p>
      We're testing that xref uses the <code class="tag-element">titleabbrev</code>
      element where appropriate.
    </p>
                  <p>
      Article: <a href="#article" title="Unit Test: xref.003.xml">Unit 03</a>.
    </p>
                  <p>
      Section: <a href="#section" title="Cross-reference targets">Section 2, “Cross-ref targets”</a>.
    </p>
                  <p>
      Figure: <a href="#figure" title="Long Title for Figure">Figure 1, “Short Fig Title”</a>.
    </p>
                  <p>
      Example: <a href="#example" title="Long Title for Example">Example 1, “Short Examp Title”</a>.
    </p>
                  <p>
      Table: <a href="#table" title="Long Title for Table">Table 1, “Short Table Title”</a>.
    </p>
               </div>
            </section>
            <section id="section" class="section">
               <div class="section-titlepage">
                  <h2>2. Cross-reference targets</h2>
               </div>
               <div class="content">
                  <p>
      Targets for cross-reference tests.
    </p>
                  <figure id="figure" class="figure-wrapper">
                     <div class="figure">
                        <div id="figure.2" class="mediaobject">
                           <div class="longdesc-link">[<a class="dialog-link" href="#longdesc-xxx" title="long description link">D</a>]</div>
                           <p>Bit of text</p>
                        </div>
                     </div>
                     <div class="title">Figure 1. Long Title for Figure</div>
                  </figure>
                  <figure id="example" class="example-wrapper">
                     <div class="title">Example 1. Long Title for Example</div>
                     <div class="example">
                        <p>Substance of example</p>
                     </div>
                  </figure>
                  <figure id="table" class="table-wrapper">
                     <div class="title">Table 1. Long Title for Table</div>
                     <div class="table">
                        <table border="0" summary="Table 1. Long Title for Table" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                           <colgroup>
                              <col class="tcol1" width="100%"/>
                           </colgroup>
                           <tbody>
                              <tr>
                                 <td>table cell</td>
                              </tr>
                           </tbody>
                        </table>
                     </div>
                  </figure>
               </div>
            </section>
         </div>
         <section deltaxml:deltaV2="A" id="section" class="section">
            <div class="section-titlepage">
               <h2>2. Cross-reference targets</h2>
            </div>
            <p>
      Targets for cross-reference tests.
    </p>
            <figure id="figure" class="figure-wrapper">
               <div class="figure">
                  <div id="figure.2" class="mediaobject">
                     <div class="longdesc-link">[<a class="dialog-link" href="#longdesc-xxx" title="long description link">D</a>]</div>
                     <p>Bit of text</p>
                  </div>
               </div>
               <div class="title">Figure 1. Long Title for Figure</div>
            </figure>
            <figure id="example" class="example-wrapper">
               <div class="title">Example 1. Long Title for Example</div>
               <div class="example">
                  <p>Substance of example</p>
               </div>
            </figure>
            <figure id="table" class="table-wrapper">
               <div class="title">Table 1. Long Title for Table</div>
               <div class="table">
                  <table border="0" summary="Table 1. Long Title for Table" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                     <colgroup>
                        <col class="tcol1" width="100%"/>
                     </colgroup>
                     <tbody>
                        <tr>
                           <td>table cell</td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A=B" class="longdesc-list">
            <div id="longdesc-xxx" class="dialog longdesc-hide">
               <p>Bit of text</p>
            </div>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="../../resources/js/annotation.js"/>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>