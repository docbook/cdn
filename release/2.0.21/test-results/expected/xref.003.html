<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: xref.003.xml</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="article" class="article"><header class="article-titlepage"><h2>Unit Test: xref.003.xml</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div><p class="CVS">$Id: xref.003.xml 4056 2004-11-23 13:28:50Z nwalsh $</p></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#xreftests">1. Xref test: titleabbrev</a></span></li><li><span><a href="#section">2. Cross-ref targets</a></span></li></ul></div></div><section id="xreftests" class="section"><div class="section-titlepage"><h2>1. Xref test: titleabbrev</h2></div><p>
      We're testing that xref uses the <code class="tag-element">titleabbrev</code>
      element where appropriate.
    </p><p>
      Article: <a href="#article" title="Unit Test: xref.003.xml">Unit 03</a>.
    </p><p>
      Section: <a href="#section" title="Cross-reference targets">Section 2, “Cross-ref targets”</a>.
    </p><p>
      Figure: <a href="#figure" title="Long Title for Figure">Figure 1, “Short Fig Title”</a>.
    </p><p>
      Example: <a href="#example" title="Long Title for Example">Example 1, “Short Examp Title”</a>.
    </p><p>
      Table: <a href="#table" title="Long Title for Table">Table 1, “Short Table Title”</a>.
    </p></section><section id="section" class="section"><div class="section-titlepage"><h2>2. Cross-reference targets</h2></div><p>
      Targets for cross-reference tests.
    </p><figure id="figure" class="figure-wrapper"><div class="figure"><div id="figure.2" class="mediaobject"><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d46975e88" title="long description link">D</a>]</div><p>Bit of text</p></div></div><div class="title">Figure 1. Long Title for Figure</div></figure><figure id="example" class="example-wrapper"><div class="title">Example 1. Long Title for Example</div><div class="example"><p>Substance of example</p></div></figure><figure id="table" class="table-wrapper"><div class="title">Table 1. Long Title for Table</div><div class="table"><table border="0" summary="Table 1. Long Title for Table" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="100%" /></colgroup><tbody><tr><td>table cell</td></tr></tbody></table></div></figure></section><div class="longdesc-list"><div id="longdesc-d46975e88" class="dialog longdesc-hide"><p>Bit of text</p></div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></body></html>